bundles/binding/org.openhab.binding.zwave/database/telldus/tzdw-100.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+  <Model>Telldus</Model>
+  <Label lang="en">TZDW-100</Label>
+  <CommandClasses>
+    <Class><id>0x5e</id></Class>
+    <Class><id>0x5a</id></Class>
+    <Class><id>0x7a</id></Class>
+    <Class><id>0x85</id></Class>
+    <Class><id>0x8e</id></Class>
+    <Class><id>0x59</id></Class>
+    <Class><id>0x70</id></Class>
+    <Class><id>0x80</id></Class>
+    <Class><id>0x72</id></Class>
+    <Class><id>0x71</id></Class>
+    <Class><id>0x73</id></Class>
+    <Class><id>0x30</id></Class>
+    <Class><id>0x86</id></Class>
+    <Class><id>0x84</id></Class>
+    <Class><id>0x27</id></Class>
+    <Class><id>0x20</id></Class>
+    <Class><id>0x25</id></Class>
+    <Class><id>0x26</id></Class>
+  </CommandClasses>
+  <Associations>
+    <Group>
+      <Index>1</Index>
+      <Maximum>1</Maximum>
+      <Label lang="en">Lifeline: The sensor will send it's status to the node in this group whenever the status of the sensorchanges.</Label>
```

Comment:
Please break these labels into shorter names as they won't render very nicely in the UI. The label should be very short - in this case, just use 'Lifeline'. Then move the explanation into a separate Help element.

Same for the next associations...